orchagent/bufferorch.cpp

```diff
@@ -771,18 +772,26 @@ task_process_status BufferOrch::processPriorityGroup(Consumer &consumer)
             }
             if (port.m_priority_group_lock[ind])
             {
-                SWSS_LOG_WARN("Priority group %zd on port %s is locked, will retry", ind, port_name.c_str());
-                return task_process_status::task_need_retry;
+                SWSS_LOG_WARN("Priority group %zd on port %s is locked, pending profile 0x%" PRIx64 " until unlocked", ind, port_name.c_str(), sai_buffer_profile);
```

Comment:
@stephenxs  can you please rephrase the message? are you waiting for the profile to be configured? what does it mean profile unlock?

Reply:
There is a lock protecting the profile of the PG from being modified when the PG is under the PFC storm. Because the zero buffer profile has been applied to handle the PFC storm. During this time the new profile will be pending.
When the PFC storm situation has gone the lock is released and then the pending profile will be applied. This is what I mentioned unlock.
I think the message is clear. Let me know if you still think it’s unclear. @liat-grozovik .
Thanks.